04/app.js

```diff
@@ -2,8 +2,19 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
+        day: '28',
+        month: '07',
         year: '1985'
     }
+}
+
+const birthday = Object.values(user.born).slice(0, 2).map(el => parseInt(el));
+
+checkBirthday(birthday);
+
+function checkBirthday(arr) {
+    const today = new Date();
+    (today.getDate() - arr[0] === 0 && (today.getMonth() + 1) - arr[1] === 0) ?
```

Comment:
Tutaj kompletnie jest nie czytelne co to jest `arr[0]` oraz `arr[1]`.
Przyznam, że to rozwiązanie jest imponujące, ale czytelność jest ważniejsza.
Na pewno koledzy z pracy by docenili ten drugi aspekt - w szczególności jak będą mieli przejąć po Tobie projekt :P